django/db/backends/sqlite3/base.py

```diff
@@ -174,7 +174,9 @@ def get_connection_params(self):
                 "settings.DATABASES is improperly configured. "
                 "Please supply the NAME value.")
         kwargs = {
-            'database': settings_dict['NAME'],
+            # TODO: Python 3.7 sqlite supports pathlib. Remove call to str().
+            # https://bugs.python.org/issue33496
```

Comment:
I don't think we need the bpo link.

Perhaps:

```python
# TODO: Remove str() when dropping support for Python 3.7.
```

Reply:
Personally, I find the link is nice as it provides confidence and justification to the future maintainer that the change is correct. It answers the "why?". But I don't mind changing it. I've done so in the latest revision. I tweaked the language slightly to:

```
# TODO: Remove str() when dropping support for PY36.
```

So as to match the identifier in `django/utils/version.py` to assist future grepping.

Reply:
> Personally, I find the link is nice as it provides confidence and justification to the future maintainer that the change is correct.

Ok. I think that the link can go back in - I suggested that and then thought about removing the "sqlite supports pathlib" after.

> So as to match the identifier in `django/utils/version.py` to assist future grepping.

Good call.

Reply:
Back :slightly_smiling_face:

Reply:
Super. Just as a data point, I **do** quite like the bpo link, precisely so I can follow the reasoning. (Maybe others have more powers, but not me. 🙂)